Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.py

```diff
@@ -197,37 +214,84 @@ def __init__(self, params: Dict[str, Any], proxy, disable_ssl):
-            raise DemistoException('https proxy value is empty. Check Demisto server configuration' + str(proxies))
-        https_proxy = proxies['https']
-        if not https_proxy.startswith('https') and not https_proxy.startswith('http'):
-            https_proxy = 'https://' + https_proxy
-        parsed_proxy = urllib.parse.urlparse(https_proxy)
-        proxy_info = httplib2.ProxyInfo(
-            proxy_type=httplib2.socks.PROXY_TYPE_HTTP,  # disable-secrets-detection
-            proxy_host=parsed_proxy.hostname,
-            proxy_port=parsed_proxy.port,
-            proxy_user=parsed_proxy.username,
-            proxy_pass=parsed_proxy.password)
-    return httplib2.Http(proxy_info=proxy_info, disable_ssl_certificate_validation=disable_ssl)
+                {backoff factor} * (2 ** ({number of total retries} - 1))
+
+            seconds. If the backoff_factor is 0.1, then :func:`.sleep` will sleep
+            for [0.0s, 0.2s, 0.4s, ...] between retries. It will never be longer
+            than :attr:`Retry.BACKOFF_MAX`.
+
+            By default, backoff_factor set to 5
 
+        :type raise_on_redirect ``bool``
+        :param raise_on_redirect: Whether, if the number of redirects is
+            exhausted, to raise a MaxRetryError, or to return a response with a
+            response code in the 3xx range.
 
-def validate_response(client, url, method='GET', body=None):
+        :type raise_on_status ``bool``
+        :param raise_on_status: Similar meaning to ``raise_on_redirect``:
+            whether we should raise an exception, or return a response,
+            if status falls in ``status_forcelist`` range and retries have
+            been exhausted.
+        """
+        try:
+            method_whitelist = "allowed_methods" if hasattr(Retry.DEFAULT,  # type: ignore[attr-defined]
+                                                            "allowed_methods") else "method_whitelist"
+            whitelist_kawargs = {
+                method_whitelist: frozenset(['GET', 'POST', 'PUT'])
+            }
+            retry = Retry(
+                total=retries,
+                read=retries,
+                connect=retries,
+                backoff_factor=backoff_factor,
+                status=retries,
+                status_forcelist=status_list_to_retry,
+                raise_on_status=raise_on_status,
+                raise_on_redirect=raise_on_redirect,
+                **whitelist_kawargs  # type: ignore[arg-type]
+            )
+            http_adapter = HTTPAdapter(max_retries=retry)
+
+            if not self.disable_ssl:
+                https_adapter = http_adapter
+            elif IS_PY3 and PY_VER_MINOR >= 10:
+                https_adapter = SSLAdapter(max_retries=retry, verify=not self.disable_ssl)  # type: ignore[arg-type]
+            else:
+                https_adapter = http_adapter
+
+            self.http_client.mount('https://', https_adapter)
+
+        except NameError:
+            pass
```

Comment:
I see that this method is copied from the `baseClient` class in the `commonServerPython` file. Could we just inherit it? (`class Client(BaseClient):`)

Please feel free to reach out to me with any questions 
I'm available here or on slack.